data_validation/__main__.py

```diff
@@ -377,6 +377,10 @@ def main():
         print(run_raw_query_against_connection(args))
     elif args.command == "validate":
         validate(args)
+    elif args.command == "deploy":
```

Comment:
Let's document that this is a beta CLI option and that the env var 'PORT' must be set. I also added an [example JSON config](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/functions#json-configuration) input to the Flask app if you want to copy that

Reply:
done